test-project/mock-project/src/main/resources/META-INF/services/com.exasol.adapter.AdapterFactory

```diff
@@ -0,0 +1 @@
+com.exasol.adapter.document.moc.FixedDataAdapterFactory
```

Comment:
```suggestion
com.exasol.adapter.document.mock.FixedDataAdapterFactory
```

Do you mean `mock`? Then you will also need to rename the package in the java sources.